src/Symfony/Component/Messenger/Command/FailedMessagesRemoveCommand.php

```diff
@@ -60,29 +61,37 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $receiver = $this->getReceiver();
 
         $shouldForce = $input->getOption('force');
-        $this->removeSingleMessage($input->getArgument('id'), $receiver, $io, $shouldForce);
+        $ids = $input->getArgument('id');
+        $shouldDisplayMessages = $input->getOption('show-messages') || 1 === \count($ids);
+        $this->removeSingleMessage($ids, $receiver, $io, $shouldForce, $shouldDisplayMessages);
 
         return 0;
     }
 
-    private function removeSingleMessage(string $id, ReceiverInterface $receiver, SymfonyStyle $io, bool $shouldForce)
+    private function removeSingleMessage(array $ids, ReceiverInterface $receiver, SymfonyStyle $io, bool $shouldForce, bool $shouldDisplayMessages): void
```

Comment:
I think the method name is now misleading as it gets an array of ids as argument. What about `removeMessages(...)`?

Reply:
:+1: